packages/astro/test/fixtures/rewrite-trailing-slash-never/src/pages/foo.astro

```diff
@@ -1,3 +1,3 @@
 ---
-return Astro.rewrite("/")
+return Astro.rewrite("/base")
```

Comment:
This was an incorrect rewrite. The bug uncovered this, and I changed it. The fixture contains a `base` which means doing `/` should result to a 404 instead.